web/crux/src/app/agent/agent.service.ts

```diff
@@ -235,6 +235,7 @@ export default class AgentService {
     if (status === NodeConnectionStatus.UNREACHABLE) {
       this.logger.log(`Left: ${agent.id}`)
       this.agents.delete(agent.id)
+      this.agent_count.dec()
```

Comment:
Hmmm 🤔  nice!

Reply:
Camel case? 😊

Reply:
For the name of the variable?

Reply:
i am sure you can set something in the decorator and use whatever variable name u want